sdk/core/aborter/tsconfig.json

```diff
@@ -0,0 +1,39 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
+    "target": "es5" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
+    "module": "es6" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
+    "lib": [] /* lib dependencies are triple-slash directives in lib/index.ts */,
+    "declaration": true /* Generates corresponding '.d.ts' file. */,
+    "declarationMap": true /* Generates a sourcemap for each corresponding '.d.ts' file. */,
+    "sourceMap": true /* Generates corresponding '.map' file. */,
+    "outDir": "./dist-esm" /* Redirect output structure to the directory. */,
+    "stripInternal": true /* Do not emit declarations for code with @internal annotation*/,
+    "declarationDir": "./typings" /* Output directory for generated declaration files.*/,
```

Comment:
I believe we now prefer `types` over `typings`.

Reply:
Updated to use `types` instead of `typings`.